collections/front-end-javascript-frameworks/index.md

```diff
@@ -1,27 +1,14 @@
 ---
 items:
- - marko-js/marko
  - mithriljs/mithril.js
  - angular/angular
  - emberjs/ember.js
- - knockout/knockout
- - tastejs/todomvc
- - spine/spine
  - vuejs/vue
- - Polymer/polymer
- - facebook/react
```

Comment:
Hm.. I'm not sure whether a reader here would understand why Angular would be considered a framework but React would not. (I also don't get it, but happy to learn!)

I'm all for seeing these organised better, but I think in that case we should also explain the terms involved (e.g. "framework" and whatever we end up calling the other page) so that the end result  is not a reading picking whichever sounds more familiar to them, with the other half of the list becoming undiscovered / receiving less traffic.